xtask/src/main.rs

```diff
@@ -0,0 +1,45 @@
+use clap::{Parser, Subcommand};
+
+#[derive(Parser)]
+#[command(name = "xtask", version, about = "fastn xtask CLI")]
+struct Cli {
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand)]
+enum Commands {
+    NewApp { name: String },
```

Comment:
I don't think this command should exist as you will use fastn template to create new apps. Once the app is created from template why would we ask it to create a new app? We ask fastn to create new apps.